go/chat/storage/storage_blockengine.go

```diff
@@ -12,7 +12,7 @@ import (
 	"golang.org/x/net/context"
 )
 
-const blockIndexVersion = 6
+const blockIndexVersion = 7
```

Comment:
Yes it is. Old rows will refuse to be read. I bet they never get gc'd.
https://github.com/keybase/client/blob/master/go/chat/storage/storage_blockengine.go#L123